platform/security/docs/src/main/resources/IntContents.adoc

```diff
@@ -1426,3 +1426,58 @@ The `AnonymousInterceptor` is installed by default with DDF Security Application
 . Click save.
 
 Once these configurations have been added, the AnonymousInterceptor is ready for use. Both secure and non-secure requests will be accepted by all secure DDF service endpoints.
+
+== Security IdP
+
+The DDF Security IdP application provides service provider handling that satisfies the SAML 2.0 Web SSO profile in order to support external IdPs (Identity Providers).
```

Comment:
Should we link "SAML 2.0 Web SSO profile" to the standard ?

http://docs.oasis-open.org/security/saml/v2.0/saml-profiles-2.0-os.pdf